@xen-orchestra/lite/src/components/vm/network/VmVifsTable.vue

```diff
@@ -100,7 +100,7 @@
                 size="small"
               />
               <div v-else-if="column.id === 'status'" v-tooltip>
```

Comment:
```suggestion
              <div v-else-if="column.id === 'status'">
```
v-tooltip is useless here